ansible/setup.yml

```diff
@@ -6,14 +6,20 @@
   - name: find the ip of docker-machine
     local_action: shell "docker-machine" "ip" "{{docker_machine_name | default('whisk')}}"
     register: result
-    ignore_errors: true
+    when: "'environments/mac' in inventory_dir"
```

Comment:
related to markus' concern, if user want to create a new own mac env, it also works fine cause the ips are input in the env. Our `mac` env uses 'dynamic' IP which are generated by setup.yml. So, the `when` condition is there for good reasons.